connector/connect/src/main/protobuf/spark/connect/relations.proto

```diff
@@ -37,10 +37,11 @@ message Relation {
     Join join = 5;
     Union union = 6;
     Sort sort = 7;
-    Fetch fetch = 8;
+    Limit limit = 8;
     Aggregate aggregate = 9;
     SQL sql = 10;
     LocalRelation local_relation = 11;
+    Offset offset = 13;
```

Comment:
Why two relations?

Reply:
Please see the PR description. TLDR DataFrame does not follow SQL semantic.